### ISSUE TITLE
[TileMap]Even each tileset only have one image,still can't show correctly.(withTestCase)

### ISSUE THREAD

**Issue by zloop1982** [cocos2d/cocos2d-x#16670]
- cocos2d-x version:3.13.1
- devices test on: chrome 
- developing environments
  - NDK version:
  - Xcode version:
  - VS version:
  - browser type and version:chrome  53.0.2785.143
  - Tiled Map Editor 0.17.0

Steps to Reproduce:
1. try to load tiled map attached here:[tilemap.zip](https://github.com/cocos2d/cocos2d-x/files/519080/tilemap.zip)
   
   use this code : 
   
      `var map = new cc.TMXTiledMap(res.map_tmx);`
      `this.addChild(map, 2, TAG_TILE_MAP);`
2. run the game in browser.

NOTE: in this map , each tileset only have one image. but have 8 tilesets.

[tilemap.zip](https://github.com/cocos2d/cocos2d-x/files/519080/tilemap.zip)

**Comment by zloop1982** [comment]
Is there an limitation "one layer only use one image"?
Or, "one layer only use one tilesize"?

I split the metal,rock,wall in different layer, and then, seems better.

**Comment by zloop1982** [comment]
finally  set every tielset's tilewidth and tileheight to 64 as the map set , get correct result.

**Comment by stevetranby** [comment]
yeah there's definitely a one tileset per layer limitation in the cocos2d tilemap renderer
